src/IntroPage/IntroPage.stories.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryRouter from 'storybook-router';
+
+import IntroPage from './';
+
+storiesOf('IntroPage', module)
+  .addDecorator(StoryRouter())
+  .add('Default', () => <IntroPage />);
```

Comment:
I think you noticed that we almost never use storybook even having it installed (shame on us), we use it so little we haven't realized that storybook wasn't working until your (https://github.com/kiwicom/smart-faq/pull/210), but if we start using it more often I think we should use it to show components not "screens", even so there won't be many components since we relay in Orbit Components as ui library.  what do you think ? @jaroslav-kubicek @git-toni @kantord @edgarordonez

Reply:
Yep makes sense.. sorry for the trouble ;)

Reply:
I totally agree with @jalbertsr . Storybook seems like a great way to showcase our more "complex" components such as Accordion, Breadcrumbs, etc.

Reply:
I agree as well